modules/uncertainty/R/get.parameter.samples.R

```diff
@@ -201,10 +201,10 @@ get.parameter.samples <- function(settings,
         quantiles = 0.5
       )
     } else if (settings$ensemble$size > 1) {
-      
+      ensemble.method<-"random"
       ## subset the trait.samples to ensemble size using Halton sequence
       ensemble.samples <- get.ensemble.samples(settings$ensemble$size, trait.samples, 
-                                               env.samples, ens.sample.method, param.names)
+                                               env.samples,ensemble.method, param.names)
```

Comment:
This bit of code shouldn't be changed -- we shouldn't hard code the sampling method using a new variable. It  should still (ultimately) come from the settings